tsconfig-common.json

```diff
@@ -23,6 +23,6 @@
         "noImplicitAny": false,
         /* Completeness */
         "skipDefaultLibCheck": true, // it's safe to assume that built-in .d.ts files are correct
-        "skipLibCheck": false
+        "skipLibCheck": true
```

Comment:
I'm going to remove this change from this PR since it has to do with type checking and not linting.  When I bring it back I'll make sure to include a comment about this.